### PR TITLE
Feature/front end integration

### PR DESCRIPTION
**Pull request by BrownMark**
Regards 4.4 on the boards

### CODE REVIEW

**Review by William-Unosquare — APPROVED** [review]

File: web-app/src/components/RequestHoliday/container.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    })
+    .catch(error =>{
+        Swal('Could not complete holiday request', error.message, 'error');
+    })
+  }
+
+  requestHolidays(holidays){
+    this.HolidayService.requestHolidays(holidays)
+    .then(response =>{
+       Swal('Holiday booked', error.message, 'error');
```
Comment (BrownMark) [inline web-app/src/components/RequestHoliday/container.js]: Wrong Swal call here.

File: web-app/src/components/RequestHoliday/container.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        employee : this.props.user,
+        holidayStatusId : 1,
+        isHalfDay : this.state.halfDayChecked
+      }
+  }
+
+  requestHoliday(holiday){
+    this.HolidayService.requestHoliday(holiday)
+    .then(response =>{
+       Swal('Holiday booked', error.message, 'error');
```
Comment (BrownMark) [inline web-app/src/components/RequestHoliday/container.js]: Wrong Swal call here.

File: web-app/dist/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+    <title>Admin Core</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (BrownMark) [inline web-app/dist/index.html:6]: I would normally .gitignore the Dist folder

File: web-app/src/components/Header/index.jsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                    </Navbar.Header>
+                    <Navbar.Collapse>
+                        <Nav>
+                            <NavItem eventKey={1}  onClick={this.onHomeClick}>
+                            Home
+                            </NavItem>
+                        </Nav>
+                        <Nav pullRight>
+                            <NavItem eventKey={1}  onClick={this.onSettingsClick}
+                                style={{visibility: this.props.user.employeeRoleId < 3 ? 'visible' : 'hidden' }}>
```
Comment (BrownMark) [inline web-app/src/components/Header/index.jsx]: I'm not a huge fan of magic numbers (for maintainability), might be better using an enumeration here, especially for readability. An example could be something like:

const roles = {
  admin: 1,
  super: 2,
  standard: 3
};

This check would become this.props.user.employeeRoleId < roles.standard etc.

Don't have to do it, but I think it helps

File: web-app/src/components/RequestHoliday/container.js [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+        halfDayChecked: !this.state.halfDayChecked
+      });
+  }
+
+
+  buildHoliday(date){
+      return {
+        date : date,
+        employee : this.props.user,
+        holidayStatusId : 1,
```
Comment (BrownMark) [inline web-app/src/components/RequestHoliday/container.js]: Same again with the enumeration suggestion

File: web-app/dist/index.html (hunk 3 above)
Comment (bgribz) [reply web-app/dist/index.html:6]: Added to .gitignore :+1:

**Pull request merged**